src/Tickets/Emails/Email/RSVP.php

```diff
@@ -0,0 +1,319 @@
+<?php
+/**
+ * Class RSVP
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails\Email;
+
+use TEC\Tickets\Commerce\Settings;
```

Comment:
Do we need this here?

Reply:
Ah. No we don't. It was copied from the Ticket email class. I'll remove it.

Reply:
It was added back, now that I see that we want to use the original setting from Commerce Settings as the default.